app/views/content/is-teaching-right-for-me/become-an-early-years-teacher.md

```diff
@@ -47,7 +47,7 @@ calls_to_action:
   EY-self-funded:
```

Comment:
This is a comment for line 43, value needs changing from Cost: "Up to £9,250" to Cost: Up to eycoursefeesundergrad

Have left the $ off the above value as it's messing with the formatting in the comment

There's no way to add this to line 43 that I can see, any ideas @gemmadallmandfe @martyn-w ?

Reply:
You can edit the file directly (three dots on top right) and then commit your edit :-)